packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -894,6 +908,86 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
         return dragValue;
     };
 
+    /**
+     * Handles touch dragging
+     */
+    private handleUpperThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.upperThumb);
+    };
+
+    private handleLowerThumbTouchStart = (e: React.TouchEvent): void => {
+        this.handleThumbTouchStart(e, SliderThumb.lowerThumb);
+    };
+
+    private handleThumbTouchStart = (e: React.TouchEvent, thumb: SliderThumb): void => {
+        if (e.defaultPrevented || this.isBusyOrDisabled()) {
+            return;
+        }
+
+        e.preventDefault();
+        (e.target as HTMLElement).focus();
+        window.addEventListener("touchend", this.handleTouchEnd);
+        window.addEventListener("touchcancel", this.handleTouchEnd);
+        window.addEventListener("touchmove", this.handleTouchMove);
+        this.setState({
+            isTouchDragging: true,
+            activeThumb: thumb,
+        });
+        const thisTouch: Touch = e.nativeEvent.touches.item(0);
+        this.updateDragValue(this.getDragValue(thisTouch, thumb), thumb);
+    };
+
+    /**
+     *  Returns first valid touch found in a touch event
+     */
+    private getValidTouch = (event: TouchEvent): Touch => {
```

Comment:
this method *can* return `null` - the type signature should reflect that.

Reply:
Really?  I guess my c# single return type background has me expecting that any function that returns an object could return null.  But sure.  I've probably done this a bunch...

Reply:
Yea it's not illegal in TypeScript either (which is why this doesn't cause an error) but TypeScript does expose a `strictNullChecks` flag on the compiler to avoid that behavior. I'd like to get us there so our type system is more sound but there are quite a few issues to fix